src/solvers/refinement/string_refinement.cpp

```diff
@@ -283,107 +283,6 @@ static void substitute_function_applications_in_equations(
     eq.rhs() = substitute_function_applications(eq.rhs(), generator);
 }
 
-/// For now, any unsigned bitvector type of width smaller or equal to 16 is
```

Comment:
Please fix the typo in the commit message `string_refiniment_util`

Reply:
fixed